dbms/src/Storages/DeltaMerge/ColumnFile/ColumnFileTiny.cpp

```diff
@@ -327,15 +327,15 @@ PageIdU64 ColumnFileTiny::writeColumnFileData(
 
     auto data_size = write_buf.count();
     if (const auto & file_provider = dm_context.global_context.getFileProvider();
-        file_provider->isKeyspaceEncryptionEnabled())
+        unlikely(file_provider->isEncryptionEnabled(dm_context.keyspace_id)))
```

Comment:
Why this is `unlikely`?

Reply:
to reduce the performance impact on keyspaces that do not enable encryption.

Reply:
So customers tend to not use the encryption feature?

Reply:
When a tenant enables encryption, which means the customer can suffer performance impact. But for those tenants who do not enable encryption, we should reduce the performance impact as possible.